habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -199,14 +215,125 @@ class TeleportActionConfig(ActionConfig):
+    The waypoint is specified in the frame of the robot.
+    Suitable for differential drive robots.
+    """
+    type: str = "WaypointAction"
+    action_duration: float = 1.0  # seconds
+    # Action space range
+    waypoint_lin_range: List[float] = field(
+        default_factory=lambda: [-0.5, 0.5]
+    )  # meters
+    waypoint_ang_range: List[float] = field(
+        default_factory=lambda: [-np.pi, np.pi]
+    )  # radians
+    delta_ang_range_camera_pitch: List[float] = field(
+        default_factory=lambda: [-0.2, 0.2]
+    )  # radians
+    wait_duration_range: List[float] = field(
+        default_factory=lambda: [0.0, 10.0]
+    )  # seconds
+    yaw_input_in_degrees: bool = False
+    # Early stopping criteria
     min_abs_lin_speed: float = 0.025  # meters/sec
-    min_abs_ang_speed: float = 1.0  # # deg/sec
-    time_step: float = 1.0  # seconds
+    min_abs_ang_speed: float = 0.018  # rad/sec (1 deg/sec)
+    min_abs_ang_speed_camera_pitch: float = 0.018  # rad/sec (1 deg/sec)
+    # Controller parameters
+    v_max: float = 0.3
+    w_max: float = 0.45
+    w_max_camera_pitch: float = 0.90
+    acc_lin: float = 0.2
+    acc_ang: float = 0.6
+    acc_ang_camera_pitch: float = 0.6
+    max_heading_ang: float = np.pi / 4  # rad
+    lin_error_tol: float = 0.01
+    ang_error_tol: float = 0.025
+
+
+@dataclass
+class MoveForwardWaypointActionConfig(WaypointControlActionConfig):
+    r"""
+    In Navigation tasks only, this discrete action will move the robot forward by
+    a fixed amount determined by the SimulatorConfig.forward_step_size amount.
+    This is implemented by generating a WaypointAction with predefined parameters.
+    """
+    type: str = "MoveForwardWaypointAction"
+    max_wait_duration: float = 3.0  # seconds
+    forward_step_size: float = 0.25  # meters
+
+
+@dataclass
+class TurnLeftWaypointActionConfig(WaypointControlActionConfig):
+    r"""
+    In Navigation tasks only, this discrete action will rotate the robot to the left
+    by a fixed amount determined by the SimulatorConfig.turn_angle amount.
+    This is implemented by generating a WaypointAction with predefined parameters.
+    """
+    type: str = "TurnLeftWaypointAction"
+    max_wait_duration: float = 3.0  # seconds
+    turn_angle: float = np.pi / 6  # rad (30 degrees)
+
+
+@dataclass
+class TurnRightWaypointActionConfig(WaypointControlActionConfig):
+    r"""
+    In Navigation tasks only, this discrete action will rotate the robot to the right
+    by a fixed amount determined by the SimulatorConfig.turn_angle amount.
+    This is implemented by generating a WaypointAction with predefined parameters.
+    """
+    type: str = "TurnRightWaypointAction"
+    max_wait_duration: float = 3.0  # seconds
+    turn_angle: float = np.pi / 6  # rad (30 degrees)
+
+
+@dataclass
+class LookUpDiscreteToVelocityActionConfig(WaypointControlActionConfig):
+    r"""
+    In Navigation tasks only, this discrete action will rotate the robot to the left
```

Comment:
Is the doc string, `In Navigation tasks only, this discrete action will rotate the robot to the left`, correct?